src/controllers/userController.ts

```diff
@@ -0,0 +1,112 @@
+import { Request, Response } from "express";
+import { ICreateUser } from "../helpers/dto";
+import { userService } from "../services/userService";
+const userServiceInstance = new userService();
+
+export class userController {
+  async createUser(request: Request, response: Response): Promise<Response> {
+    try {
+      const {
+        username,
+        firstname,
+        lastname,
+        email,
+        password,
+        position,
+        age,
+        gender,
+      }: ICreateUser = request.body;
+      const user = await userServiceInstance.createUser({
+        username,
+        firstname,
+        lastname,
+        email,
+        password,
+        position,
+        age,
+        gender,
+      });
+      return response.status(201).send(user);
```

Comment:
For legibility reasons, I'd suggest you to use this lib https://www.npmjs.com/package/http-status-codes